pegasus/sites.v3/code.org/public/teach.haml

```diff
@@ -19,7 +19,7 @@ theme: responsive_full_width
       %img.rounded-corners.box-shadow-primary{src: "/images/teach-page-top.png", alt: "", style: "position: relative; width: 100%;"}
     .clear
 
-= view :ai_pl_skinny_banner
+= view :dance_party_ai_skinny_banner
```

Comment:
Added the new banner here:

<img width="1084" alt="Teach page" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/7e1fc106-2174-4d70-b85f-b7de70900227">